src/api/context.ts

```diff
@@ -43,6 +44,9 @@ export namespace ChectlContext {
     ctx[IS_OPENSHIFT] = await kube.isOpenShift()
     ctx[IS_OPENSHIFT4] = await kube.isOpenShift4()
 
+    ctx.isChectl = getProjectName() === 'chectl'
+    ctx.isNightly = getProjectVersion().includes('next') || getProjectVersion() === '0.0.2'
```

Comment:
it looks strange to have nightly when version is 0.0.2

Reply:
Well, we still have version `0.0.2` hardcoded in `package.json`, so if build `chectl` from source code, resulting 'binary' will always have version `0.0.2`. I decided to mark built from source version as nightly, because it is often used in development.